### ISSUE TITLE
Property 'type' does not exist on type 'WebhookEvent'

### ISSUE THREAD

**Issue by newsbielt703** [line/line-bot-sdk-nodejs#164]
```js
function handleEvent(event: WebhookEvent) {
    if (event.type === "message") { //error
      return handleReply(event.replyToken);
    }
}
```

After type `LINEThingsScenarioExecutionEvent` became a possible type of `WebhookEvent`, I can't check the type of event before taking a reply token.

**Comment by xingoxu** [comment]
https://github.com/line/line-bot-sdk-nodejs/pull/158
I think it's this.
I'll urge my workmate to review the pr and release a new package.

**Comment by xingoxu** [comment]
Released in v6.8.2
